service_discovery/samples/simple-discovery/consumer/src/main.rs

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+
+//! A simple consumer for a sample of Chariott Service Discovery.
+//!
+//! This consumer "discovers" the hello world service through Chariott, and then
+//! directly calls the SayHello method on it, using a known interface. This returns
+//! a message containing "Hello, " followed by the string provided in the request.
+
+// Tells cargo to warn if a doc comment is missing and should be provided.
+#![warn(missing_docs)]
+
+use proto_servicediscovery::chariott_registry::v1::registry_client::RegistryClient;
+use proto_servicediscovery::chariott_registry::v1::DiscoverServiceRequest;
+use proto_servicediscovery::hello_world::v1::hello_world_client::HelloWorldClient;
+use proto_servicediscovery::hello_world::v1::HelloRequest;
+use tonic::Request;
+use tracing::info;
+use tracing_subscriber::util::SubscriberInitExt;
+use tracing_subscriber::EnvFilter;
+
+/// URL for the chariott service registry
+const CHARIOTT_SERVICE_REGISTRY_URL: &str = "http://0.0.0.0:50000";
```

Comment:
Please move this to a config file.

Reply:
I have created this issue to add configuration in a follow-up change: https://github.com/eclipse-chariott/chariott/issues/144